### PR TITLE
Adding a down-arrow on the general graphics dialog on Plot Options button

### LINKED ISSUE #8652: Adding a down-arrow on the general graphics dialog?
@MeSophie I'd like your opinion on this proposed change and I'd probably like to ask someone else - maybe David or Lily too, before this change is made.  But is you agree with Patrick then we may continue.

Here isw the current dialog.

image

I am really happy with the layout.  But @Patowhiz has questioned the unusual set of buttons towards to bottom.  He wonders why we don't (instead) go back to your wider (Plot Options) button and add the Down arrow there - as we do in many other graph dialogs.

Now it isn't quite the same, because in the other dialogs we have different geoms in the down arrow there.  Here would would have Facet, Titles, X-Axis, Y-Axis, Themes, Colour.  And nopw we might as well have the full list, so Facets, Layers, Titles, X-Axis, Y-Axis, Themes, Coordinates, Colour, Annotation.

This suggestion is growing on me.  Sophie, what do you think?

### PR DESCRIPTION
**Pull request by MeSophie**
Fixes #8652
@rdstern @Patowhiz  I added a down-arrow on the general graphics dialog. Please have a look.

### CODE REVIEW

**Review by rdstern — COMMENTED** [review]
@MeSophie great.  Can you add layers after facets, in the list, so it is the same as the tabs in the sub-dialog.
And you need to return the control to its previous size so the French translation works!  Then we could merge!

**Comment by MeSophie** [comment]
@rdstern Please check again the change.

**Comment by MeSophie** [comment]
@rdstern Please can you check again the change.

**Review by rdstern — COMMENTED** [review]
@MeSophie it is just a detail but plot options is still narrower than before - and I thought you said the width was needed for the French?

image

**Comment by MeSophie** [comment]
@rdstern I think the size of plot options button is okay now.

**Review by rdstern — DISMISSED** [review]
Perfect.  @N-thony can you review and then merge?

**Review by rdstern — DISMISSED** [review]
Looks fine now.  I like the down arrow, which makes the dialog nice and clean.  @N-thony the conflicts are resolved, so back to you.

File: instat/sdgPlots.vb [hunk 1]
```diff
@@ -3337,2204 +3337,8 @@ Public Class sdgPlots
...
-    End Sub
-
-    Private Sub ucrInputCanvasFillPalette_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrInputCanvasFillPalette.ControlValueChanged
-        HideShowCanvaFill()
-        GroupeColorScale()
-    End Sub
-
-    Private Sub ucrInputFillFunction_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrInputFillFunction.ControlValueChanged
-        HideShowCanvaFill()
-        GroupeColorScale()
```
[the remark this answers is not in the document]
Comment (MeSophie) [reply instat/sdgPlots.vb:5538]: I was solving conflict. But I update the branch it is okay now.

**Comment by MeSophie** [comment]
@rdstern  Please can you test the change once more?

**Comment by MeSophie** [comment]
I replace this PR by #8697

**Pull request closed without merging**